### PR TITLE
FEATURE: added query param in standup display page

### PR DESCRIPTION
**Pull request by sahsisunny**
### What is the change?

1. `__tests__/standup/standup.test.js`: Added test cases for the standup page.
2. `standup/script.js`:  Added feature to add query parameters to the standup page, allowing the display of a user's standup.

### Test Code
image

### \*Dev Tested?

- [x] Yes
- [ ] No

### After Change Screenshots
image

##### Note for Reviewer: In the PR I fix the style of the tooltip and sidebar

### CODE REVIEW

**Review by MayurLund — APPROVED** [review]

**Comment by TanishqSingla** [comment]
Post screenshots for your change

**Comment by sahsisunny** [comment]
> Post screenshots for your change

Added

**Review by Pratiyushkumar — APPROVED** [review]
LGTM

**Review by heyrandhir — APPROVED** [review]

**Review by gauravsinhaweb — APPROVED** [review]

File: standup/script.js [hunk 1]
```diff
@@ -219,10 +219,15 @@ async function searchButtonHandler() {
   const usernames = searchInput.value
     .split(',')
     .map((username) => username.trim());
-  const filteredUsernames = [...new Set(usernames)];
+  const formattedUsernames = usernames.map((username) => `user:${username}`);
```
Comment (Aryex82) [inline standup/script.js:222]: will the current implementation allow duplicate usernames?

File: __tests__/standup/standup.test.js [hunk 2]
```diff
@@ -79,4 +79,16 @@ describe('Input box', () => {
     const loader = await page.$('.loader');
     expect(loader).toBeTruthy();
   });
+
+  it('should update the URL with the query parameter when the user writes a name', async () => {
```
Comment (Aryex82) [inline __tests__/standup/standup.test.js:83]: any tests to verify multiple query params?

**Review by Aryex82 — CHANGES_REQUESTED** [review]

File: standup/script.js (hunk 1 above)
Comment (sahsisunny) [reply standup/script.js:222]: No

File: __tests__/standup/standup.test.js (hunk 2 above)
Comment (sahsisunny) [reply __tests__/standup/standup.test.js:83]: Not needed

Comment (sahsisunny) [reply __tests__/standup/standup.test.js:83]: image

**Review by Aryex82 — APPROVED** [review]

**Pull request merged**